blueprint/abac/client-devices.md

```diff
@@ -0,0 +1,8010 @@
+Set-RuleOption -FilePath $WDACPolicy -Option 19 # Enable Dynamic Code Security
+```
+5. Deploy the file locally in audit mode and validate no additional files require whitelisting
+6. Using an administrative PowerShell session run the following to switch the policy into enforce mode
+```powershell
+Set-RuleOption -FilePath $WDACPolicy -Option 3 -Delete
+```
+7. Deploy the file locally in enforce mode and validate no additional files require whitelisting
+8. [Deploy the file globally](/blueprint/abac/client-devices.html#wdac-policy---deployment) 
+
+
+### WDAC Policy - Supplementary Policy
+
+Supplementary policies expand on the base policy and allow for whitelisting to be targeted to users and/or groups. Supplementary pollcies can contain both allow and deny rules. A supplementary policy is a base policy until it is linked as a supplementary policy to another base policy. 
+
+Supplementary policies will not work on machines pre 1903. If you are deploying to pre 1903 machines they must be merged into the base policy. [Merge policy](/blueprint/abac/hybrd-client-devices.html#wdac-policy---whitelisting-an-application) instructions are available in the hybrid client whitelisting section.
+
+#### Whitelisting an application
+
+Whitelisting of an application utilising WDAC can be completed utilising a number of methods including Hash (ACSC recommended), file path, and certificate based. Depending on the application, the chosen method of whitelisting may change. Hash whitelisting offers the highest degree of security however it increases the management overhead associated with whitelisting. 
+
+If the managed installer option within the base policy is enabled, whitelisting of applications deployed via the managed installer is not required. 
+
+![WDAC Whitelisting Application policy](/assets/images/WDAC_Intune.png)
+
+Post identification of the appropriate whitelisting method, the following procedure is used to whitelist the application:
+
+1.  Within an administrative PowerShell session navigate to the install directory of the application
+2.  Run the following command
+```powershell
+$whitelistlevel = the chosen method of whitelist e.g. hash
+$Outputlocation = the path to output the policy file
+$fallbacklevel = the backup method of whitelist e.g. publisher
+$directory = the path to be scanned
+New-CIPolicy -Level $whitelistlevel -FilePath $Outputlocation -Fallback $fallbacklevel  -ScanPath $directory -UserPEs
+```
+3. Convert the new base policy to a supplementary policy.
+
+#### Whitelisting all errors in the event log
+
+Whitelisting of an applications blocked or audited by WDAC can be completed utilising PowerShell. This is normally conducted when testing policies as it is important to test a sample of individual application functionality. 
+
+Post identification of the appropriate whitelisting method, the following procedure is used to whitelist the applications identified in the event log:
```

Comment:
Not entirely sure I understand this. Is it about blocked applications being logged? If so, suggest rephrasing to something like:

`Applications that have not yet been whitelisted are logged in the event log on execution. This can occur while trialing a new application or on updates to existing applications. To whitelist the desired application, do the following:`

Is the following important to keep? `Post identification of the appropriate whitelisting method`

Reply:
Addressed